### ISSUE TITLE
Gensever timeouts on Erlang 24 when adding a consensus group

### ISSUE THREAD

**Issue by queer** [skirino/raft_fleet#7]
When calling `RaftFleet.add_consensus_group/3`, I get timeouts on Erlang 24 but not 23. Erlang 24 gives the following error:
```
{:error,
  {:timeout,
    {GenServer, :call,
      [
        {RaftFleet.Manager, :node@hostname},
        {:await_completion_of_adding_consensus_group, :"consensus-group-name"},
        5000
      ]
    }
  }
}
```

The exact same code works on Erlang 23. Elixir version doesn't seem to matter; 1.11 and 1.12 both work as expected.

**Comment by skirino** [comment]
Thank you for filing this issue. The bug was fixed in v0.10.2 release; could you give it a try?

**Comment by queer** [comment]
Works perfectly! Thanks for the quick fix :D
